tests/BenchmarkDotNet.Tests/Environments/ProcessorBrandStringTests.cs

```diff
@@ -30,7 +31,24 @@ public class ProcessorBrandStringTests
         [InlineData("Intel(R) Core(TM) i7-5775R CPU @ 3.30GHz", "Intel Core i7-5775R CPU 3.30GHz (Max: 3.40GHz) (Broadwell)", 3.4)]
         public void CoreIsPrettifiedWithDiffFrequencies(string originalName, string prettifiedName, double actualFrequency)
         {
-            Assert.Equal(prettifiedName, ProcessorBrandStringHelper.Prettify(originalName, Frequency.FromGHz(actualFrequency)));
+            Assert.Equal(prettifiedName, ProcessorBrandStringHelper.Prettify(new CpuInfo(originalName, nominalFrequency: Frequency.FromGHz(actualFrequency)), includeMaxFrequency: true));
+        }
+        
+        [Theory]
+        [InlineData("AMD Ryzen 7 2700X Eight-Core Processor", "AMD Ryzen 7 2700X (Max: 4.10GHz)", 4.1, 8, 16)]
+        [InlineData("AMD Ryzen 7 2700X Eight-Core Processor", "AMD Ryzen 7 2700X Eight-Core Processor (Max: 4.10GHz)", 4.1, null, null)]
```

Comment:
Could we replace `AMD Ryzen 7 2700X (Max: 4.10GHz)` by `AMD Ryzen 7 2700X 4.10GHz`?